Sources/Testing/Running/Configuration.swift

```diff
@@ -20,6 +20,80 @@ public struct Configuration: Sendable {
   /// Whether or not to parallelize the execution of tests and test cases.
   public var isParallelizationEnabled = true
 
+  /// A type describing whether or not, and how, to iterate a test plan
+  /// repeatedly.
+  ///
+  /// When a ``Runner`` is run, it will run all tests in its corresponding
+  /// ``Runner/Plan`` according to the policy described by its
+  /// ``Configuration/iterationPolicy-swift.property`` property. For instance,
+  /// if that property is set to:
+  ///
+  /// ```swift
+  /// .repeating(.untilIssueRecorded, count: 10)
+  /// ```
+  ///
+  /// The entire test plan will be run repeatedly, up to 10 times. If an issue
+  /// is recorded, the current iteration will complete, but no further
+  /// iterations will be attempted.
+  ///
+  /// If the value of an instance's ``count`` property is `1`, the value of its
+  /// ``continuationCondition-swift.property`` property has no effect.
+  public struct IterationPolicy: Sendable {
+    /// An enumeration describing the conditions under which test iterations
+    /// should continue.
+    public enum ContinuationCondition: Sendable {
+      /// The test plan should continue iterating until an unknown issue is
+      /// recorded.
+      ///
+      /// When this continuation condition is used and an issue is recorded, the
+      /// current iteration will complete, but no further iterations will be
+      /// attempted.
+      case untilIssueRecorded
+
+      /// The test plan should continue iterating until an iteration completes
+      /// with no unknown issues recorded.
+      case whileIssueRecorded
+    }
+
+    /// The conditions under which test iterations should continue.
+    ///
+    /// If the value of this property is `nil`, a test plan will be run
+    /// ``count`` times regardless of whether or not issues are encountered
+    /// while running.
+    public var continuationCondition: ContinuationCondition?
+
+    /// The maximum number of times the test run should iterate.
+    ///
+    /// - Precondition: The value of this property must be greater than or equal
+    ///   to `1`.
+    public var count: Int {
```

Comment:
Can the name of this property include the word "maximum", e.g. `maximumRepetitionCount`? Just because, per the continuation conditions, it may not actually repeat the precise number of times this property returns

Reply:
Sure, I can change it.